### PR TITLE
Make compatible with ftw.theming 2.0.0

### PR DESCRIPTION
**Pull request by bierik**
⚠️ Requires https://github.com/4teamwork/ftw.theming/pull/85 which is released in `ftw.theming` 2.0.0.

### CODE REVIEW

File: ftw/subsite/browser/ressources/integration.ftw-mobile.theme.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #portal-languageselector-wrapper #portal-languageselector{
   display: none;
 
-  @include tablet {
+  @include screen-medium {
```
Comment (jone) [inline ftw/subsite/browser/ressources/integration.ftw-mobile.theme.scss]: Where is `screen-medium` defined?
Not in https://github.com/4teamwork/ftw.theming/pull/85 🤔

Comment (bierik) [reply ftw/subsite/browser/ressources/integration.ftw-mobile.theme.scss]: The mixin is defined here: https://github.com/4teamwork/ftw.theming/blob/master/ftw/theming/resources/scss/globals/variables.scss#L211

**Comment by bierik** [comment]
@jone ftw.theming 2.0.0 is released and tests are passing... could you have another look pls

**Comment by bierik** [comment]
@jone updated

**Pull request merged**